M2/libraries/gc/Makefile.in

```diff
@@ -75,7 +78,7 @@ CONFIGOPTIONS +=  --enable-parallel-mark
 # INSTALLTARGET = install-strip
 endif
 
-CONFIGOPTIONS += --enable-large-config --disable-gcj-support --disable-java-finalization
+CONFIGOPTIONS += --disable-gcj-support --disable-java-finalization
```

Comment:
Why not just use --enable-large-config. It's going to set MAX_HEAP_SECTS to 81920 anyways (at least on 64bit systems, 32bit programs can't access that much memory). Also it sets several other values to larger values relevant for larger memory usage.

Reply:
Hmm, maybe.  But could it really make a difference?